server/app/models/Program.java

```diff
@@ -92,8 +91,7 @@ public class Program extends BaseModel {
    */
   @WhenModified private Instant lastModifiedTime;
 
-  @ManyToMany
-  @JoinTable(name = "versions_programs")
```

Comment:
@bion Do you know if this is ok?

Reply:
A bit of background behind the change: I moved the @JoinTable declaration from Program -> Version. This isn't a required change, but felt like a more accurate representation of the relationship of the many-to-many relationship to me (e.g. a Version is a container for many Programs).

Relevant javadocs: https://docs.oracle.com/javaee/6/api/javax/persistence/ManyToMany.html

Reply:
@shanemc-goog to answer directly: I don't

If the tests pass though that gives me confidence that the relationships are working properly since we should be exercising them well enough